include/deal.II/matrix_free/operators.h

```diff
@@ -229,6 +230,12 @@ namespace MatrixFreeOperators
     void mult_add (LinearAlgebra::distributed::Vector<Number> &dst,
                    const LinearAlgebra::distributed::Vector<Number> &src,
                    const bool transpose) const;
+
+    /**
+     * Adjust the ghost range of the vectors to the storage requirements of
+     * the underlying MatrixFree class.
```

Comment:
maybe give a hint in documentation where that could be needed/used (i see it is in `mult_add`), in what circumstances?